edx-platform/msus-produhktiv/lms/templates/index.html

```diff
@@ -18,7 +19,7 @@
                 ${homepage_overlay_html | n, decode.utf8}
               % else:
                   ## Translators: 'Open edX' is a registered trademark, please keep this untranslated. See http://open.edx.org for more information.
-                  <h1>${Text(_(u"Welcome to the Microsoft U.S. Learning Portal"))}</h1>
+                  <h1>${Text(_(u"Welcome to the {portal_name}")).format(portal_name=configuration_helpers.get_value('PORTAL_NAME', settings.PLATFORM_NAME))}</h1>
```

Comment:
you keep PORTAL_NAME?
```suggestion
                  <h1>${Text(_(u"Welcome to the {portal_name}")).format(portal_name=configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME))}</h1>
```

Reply:
Yes, because PLATFORM NAME is being used for something else. I added it as a default so the site doesn't break.

Reply:
Ok, I leave it to your judgement